src/app/users/users.component.ts

```diff
@@ -255,7 +255,7 @@ export class UsersComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   back() {
-    this.router.navigate([ '/' ]);
+    this.router.navigate([ '../../' ], { relativeTo: this.route });

```

Comment:
Should only go up one level.

Reply:
One level does not change url for `users` page that why needed to add two level